docs-src/references/ui-configuration.md

```diff
@@ -33,7 +37,7 @@ enableOpenApi: true
 ## cors
 
 The name of the `cors` field stands for Cross-Origin Resource Sharing.
-Use this field to provide a list of domains that are authorized to access the UI Server APIs.
+Use this field to provide a [list of domains](/references/web-ui-server-env-vars#temporaladdress) that are authorized to access the UI Server APIs.
```

Comment:
"This can be set dynamically using the X env var"

But I think it is linking to the wrong spot currently - Temporal Address is the Cluster address - this is a list of domains that can make calls to the ui Server API itself.